README.md

```diff
@@ -1,75 +1,91 @@
-# DiscordBot
+# GReYBot
 A chat bot for the GReY Discord Server based on [Chalda's DiscordBot](https://github.com/chalda/DiscordBot) and [martindale's Snarl Bot](https://github.com/martindale/snarl), using [discord.js](https://github.com/hydrabolt/discord.js/)
 
+[![GitHub license](https://img.shields.io/badge/license-Apache%202-blue.svg?style=flat-square)](https://raw.githubusercontent.com/naterchrdsn/GReYBot/master/LICENSE.md)
+
+Invite the bot to your server [here.](https://discordapp.com/oauth2/authorize?client_id=283636170741514250&scope=bot&permissions=66186303)
+
+
 # Features:
-- !gif query => returns a gif example !gif cute cats doing stuff
-- !wiki query=> returns the summary of the first search result on Wikipedia
-- !wolfram query => queries Wolfram Alpha for results
-- !meme memetype "text1" "text2" => returns a meme image. notice the quotes around text, they are vitally important
-- !say text => echos text
-- !alias => create custom shorthand commands in channel!
-- !join-server => bot will join the requested server
-- @botname => responds when @mentioned
-- channel management!
+- !brew *query* => returns a summary of a specific beer or brewery
 
-And much more! Try !help to get a full list of available commands
+- !topic => passive-aggressively remind your users of the channel topic!
 
-# Installation
+- !roll 1d20 => roll for initiative! Or something else, using whatever combo of dice.
 
-This bot is written to run on top of node.js. Please see https://nodejs.org/en/download/
+- !unshorten *shortlink* => unshorten a shortened link, so you can see if it's shady or not.
+- !lmgtfy *query* => googles something for an idiot!
+- !say *text* => echos text
 
-Once you have node installed running `npm install` from the bot directory should install all the needed packages. If this command prints errors the bot won't work!
+- !prune *optional number* => prunes a number of messages from the channel.
```

Comment:
Should probably mention that this command only works if the user requesting and bot have the permission to do so.